genapkovl-lima.sh

```diff
@@ -278,7 +278,8 @@ if [ "${LIMA_INSTALL_ZSTD}" == "true" ]; then
 fi
 
 if [ "${LIMA_INSTALL_TINI}" == "true" ]; then
-    echo "tini" >> "$tmp"/etc/apk/world
+    echo tini-static >> "$tmp"/etc/apk/world
+    ln -sf /sbin/tini-static "$tmp"/usr/bin/tini
```

Comment:
should this be `/usr/sbin/tini` ?

Reply:
Maybe leave that for the regular binary, just put the "override" in /usr/bin

Reply:
Yes, I intentionally put the symlink into `/usr/bin` because it comes earlier in the `PATH` than `/sbin`. If you `apk add tini` it installs into `/sbin/tini`, so I don't want to conflict with it, just make sure the symlink is found first by `nerdctl`.

Reply:
Note that the `docker` package does the same thing; it creates a symlink from `/usr/bin/docker-init` to `/sbin/tini-static`.

I was hoping that `nerdctl` used `viper`, so that I could set `CONTAINERD_INIT_BINARY=/sbin/tini-static`, but it doesn't; it supports just a couple hand-coded environment variables.